pkg/core/handler/bind.go

```diff
@@ -0,0 +1,36 @@
+package handler
+
+import (
+	"io"
+	"net/http"
+
+	"github.com/skygeario/skygear-server/pkg/core/validation"
+)
+
+type bodyDefaulter interface {
+	SetDefaultValue()
+}
+
+type bodyValidator interface {
+	Validate() []validation.ErrorCause
+}
```

Comment:
Why don't we make these two interfaces public?

Reply:
No particular reason. I'll make it public then.